pages/express-relay/integrate-as-protocol/swaps.mdx

```diff
@@ -0,0 +1,82 @@
+import { Callout, Tabs, Steps } from "nextra/components";
+
+# How to Integrate Express Relay Swaps
+
+This guide will explain how frontends can integrate Express Relay to empower swapping.
+
+<Steps>
+### Install the Express Relay SDK
+
+Pyth provides a [Typescript SDK](https://www.npmjs.com/package/@pythnetwork/express-relay-js) to help developers integrate Express Relay into their frontends.
+
+You can install the SDK via npm or yarn. You can invoke the SDK client as below:
+
+```typescript
+import { Client } from "@pythnetwork/express-relay-js";
+
+const client = new Client(
+  { baseUrl: "https://pyth-express-relay-mainnet.asymmetric.re" },
+  undefined // Default WebSocket options
+);
+```
+
+### Request a Quote
+
+You can request a quote by calling the [`getQuote`](https://github.com/pyth-network/per/blob/281de989db887aaf568fed39315a76acc16548fa/sdk/js/src/index.ts#L501-L506) SDK method.
+
+The example below shows how you can construct a quote request for a USDC -> WSOL swap, with 100 USDC provided as input by the user:
+
+```typescript
+const userWallet = new PublicKey("<INPUT USER PUBKEY>");
+
+const quoteRequest = {
+  chainId: "solana",
+  inputTokenMint: new PublicKey("EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v"), // USDC mint
+  outputTokenMint: new PublicKey("So11111111111111111111111111111111111111112"), // WSOL mint
+  specifiedTokenAmount: {
+    side: "input",
+    amount: 100_000_000,
+  },
+  userWallet,
+};
+
+const quote = await client.getQuote(quoteRequest);
+```
+
+`quote` will contain the full details, including the amount the searcher is quoting and the transaction that the user needs to sign. It will also contain an `expirationTime`; after this time, the transaction will no longer succeed on chain, so you should request a new quote a few seconds before the `expirationTime`.
+
+### Submit User Signature to the Express Relay Server
+
+Once you show the quote to the user, the user should sign the transaction if they wish to engage in the swap. The frontend can pass this signature along to the Express Relay server, which will handle aggregating all the required signatures for the transaction and submitting it to the RPC node.
+
+Below is an example showing how the frontend can submit the signed quote transaction to the server using the [`submitQuote`](https://github.com/pyth-network/per/blob/358eedc1f9072cdfc3418fba309697580f2474f9/sdk/js/src/index.ts#L537-L542) method. The response from the `getQuote` method includes a field called `referenceId`, which the frontend should use in its submission of the user signature.
+
+```typescript
+const submitQuote = {
+  chainId: "solana",
+  referenceId: quote.referenceId,
+  userSignature: signature,
+};
+
+const txSubmitted = await client.submitQuote(submitQuote);
+```
+
+As can be seen, this will return the fully signed transaction that the server submitted to the RPC node to land on chain.
```

Comment:
```suggestion
This returns the fully signed transaction that the server submitted to the RPC node to land on chain.
```

Reply:
Or we can write "This returns the fully signed transaction that the server submitted to the RPC node."